ncm-network/src/main/pan/components/network/core-schema.pan

```diff
@@ -6,12 +6,45 @@ include 'pan/types';
 include 'quattor/functions/network';
 
 @documentation{
-    Route
+    Add route (IPv4 of IPv6)
+    Presence of ':' in any of the values indicates this is IPv6 related.
 }
 type structure_route = {
+    @{The ADDRESS in ADDRESS/PREFIX via GATEWAY}
     "address" ? type_ip
-    "netmask" ? type_ip
+    @{The PREFIX in ADDRESS/PREFIX via GATEWAY}
+    "prefix"  ? long(0..)
+    @{The GATEWAY in ADDRESS/PREFIX via GATEWAY}
     "gateway" ? type_ip
+    @{alternative notation for prefix (cannot be combined with prefix)}
+    "netmask" ? type_ip
+    @{full command used to add route (cannot be combined with other options)}
+    "command" ? string
```

Comment:
hmmm, no afaik, there are possible options that are not part of (or shouldn't be) an absolute_file_path?

Reply:
What does "full command" mean then?

Reply:
the options passed to `/sbin/ip route add`, in case you want to do some non-standard stuff
actually, we should probably sanitize the line; eg make usre there's no `;` or escaped newlnies or something bash magical...

Reply:
Ah ok, yes... probably needs to be sanitised.